tests/components/network_sediment_transporter/test_init_sediment_pulser.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+from landlab.components.network_sediment_transporter.sediment_pulser_at_links import (
+    SedimentPulserAtLinks,
+)
+from landlab.components.network_sediment_transporter.sediment_pulser_each_parcel import (
+    SedimentPulserEachParcel,
+)
+from landlab import RasterModelGrid
+
+
+def test_basic_init_each_parcel(example_nmg, example_parcels):
```

Comment:
I would like to see something other than smoke tests of the component here.

Reply:
@mcflugen the tests for the pulsers live in a separate file (test_pulse_sediment.py). I'm not sure why @keckje put the init tests in a separate file. Is that ok, as is, or do you want them combined?

Reply:
These tests now check that the class variables created by the __init__ function are present after initialization. Will this work?